README.md

```diff
@@ -2,11 +2,16 @@ KNoT Documentation
 ==================
 
 ### How to build:
+- Make sure that pip3 is installed in your machine:
+	```bash
+	$ sudo apt-get install pip3
```

Comment:
This command refers only to Debian based versions and I think the name of this package should be `python3-pip`.